packages/protocol/engine/eviction/state.go

```diff
@@ -165,35 +186,22 @@ func (s *State) Export(writer io.WriteSeeker, evictedSlot slot.Index) (err error
 // Import imports the root blocks from the given reader.
 func (s *State) Import(reader io.ReadSeeker) (err error) {
 	var rootBlockID models.BlockID
+	var commitmentID commitment.ID
 
 	return stream.ReadCollection(reader, func(i int) error {
 		if err = stream.ReadSerializable(reader, &rootBlockID, models.BlockIDLength); err != nil {
 			return errors.Wrapf(err, "failed to read root block id %d", i)
 		}
+		if err = stream.ReadSerializable(reader, &commitmentID, commitmentID.Length()); err != nil {
+			return errors.Wrapf(err, "failed to read root block id %d", i)
+		}
 
-		s.AddRootBlock(rootBlockID)
+		s.AddRootBlock(rootBlockID, commitmentID)
 
 		return nil
 	})
 }
 
-// importRootBlocksFromStorage imports the root blocks from the storage into the cache.
-func (s *State) importRootBlocksFromStorage() (importedBlocks int) {
```

Comment:
While this function is currently unused, it might come in handy for our next PR where we want to look at starting a node from disk. So maybe leave it?